create-derived/index.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
```

Comment:
You do not need to disable this ESLint rule. Just use:

```js
    return `a${count}`
```

Reply:
Sure, it was a prototype